kyo-caliban/src/main/scala/kyo/Schema.scala

```diff
@@ -17,6 +17,7 @@ given zioSchema[R, A: Flat, S](using ev: Schema[R, A], ev2: (A < S) <:< (A < (Ab
             ev.toType_(isInput, isSubscription)
 
         override def resolve(value: A < S): Step[R] =
+            given Frame = Frame.internal
```

Comment:
There are some situations where an internal frame is necessary. Frame.internal is an private[kyo] API that allows generating a frame within the kyo package.